.ui/library/flProductDetailButtonPriceSection.cpx

```diff
@@ -166,37 +166,66 @@
 			}
```

Comment:
use a flexLayout component for application titlelayout on 5 tab pages

Reply:
Miss